extensions/amp-experiment/1.0/test/test-amp-experiment.js

```diff
@@ -143,12 +149,8 @@ describes.realWin('amp-experiment', {
             'experiment-2': 'variant-d',
             'experiment-3': null,
           });
-          expectBodyHasAttributes({
```

Comment:
do we still want to test the attributes appended to body?

Reply:
Nope we dont, the body thing was just a NOOP for linting